sdk/src/main/java/com/mapbox/maps/ViewAnnotationManagerImpl.kt

```diff
@@ -1,18 +1,19 @@
-package com.mapbox.maps.viewannotation
+package com.mapbox.maps
```

Comment:
any reason to move it to another package?

Reply:
Kotlin could not see interface located in sub-package. As all internal things are generated in `com.mapbox.maps` - we need to move our class implementing this interface in same package as well.

Reply:
Thanks for clarification.